src/webview/MessageList.js

```diff
@@ -105,34 +105,32 @@ export type Props = {|
 |};
 
 /**
- * The path to the webview assets, represented as a `file:`-scheme URL.
+ * The path to the webview assets, represented as a `file:`-schema URL. This may
```

Comment:
Jargon nit: the term [is "scheme"](https://url.spec.whatwg.org/#concept-url-scheme), with an "e" at the end.